docs/next/guides/getting-started/installation.md

```diff
@@ -41,41 +86,86 @@ Run the following command in your terminal
   ```
 
   </code-block>
-  <code-block title="Nuget">
+  <code-block title="NuGet">
 
   ```bash
   PM> Install-Package Handsontable
   ```
 
   </code-block>
-  <code-block title="CDN">
+</code-group>
 
-  ```html
-  // minified JS
-  <script type="text/javascript" src="https://cdn.jsdelivr.net/npm/handsontable/dist/handsontable.full.min.js"></script>
+### Using a CDN
 
-  // minified CSS
-  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/handsontable/dist/handsontable.full.min.css" />
-  ```
+To get Handsontable's files from a CDN, use the following locations:
 
-  </code-block>
-</code-group>
+```js
+https://cdn.jsdelivr.net/npm/handsontable/dist/handsontable.full.min.js
 
-## Create a placeholder
+https://cdn.jsdelivr.net/npm/handsontable/dist/handsontable.full.min.css
+```
+
+## Import Handsontable's JavaScript
+
+Import Handsontable's JavaScript into your application.
+
+### Full distribution
```

Comment:
I think that introducing so many choices as separate sections can create a choice fatigue. In my commit 61e4709a3d856f6cb5765c7f9ebb3a8d80199480 it looked much simpler.

If you think some more subheaders could be added to my text under **Import Handsontable's JavaScript**, maybe just add two:

```
### Using CommonJS or a package manager
### Using the `script` tag
```